packages/alfa-act/package.json

```diff
@@ -11,11 +11,11 @@
     "directory": "packages/alfa-act"
   },
   "bugs": "https://github.com/siteimprove/alfa/issues",
-  "main": "src/index.js",
-  "types": "src/index.d.ts",
+  "main": "dist/index.js",
```

Comment:
**Question:** Are we going to switch to `exports` at some point? It's apparently the "modern" alternative to `main`.
https://nodejs.org/api/packages.html#package-entry-points

Reply:
Yes, probably at some point… It does restrict what external consumer can use, which is actually rather what we want :-)